{{cookiecutter.pipeline_slug}}/conf/base.config

```diff
@@ -36,4 +36,5 @@ params {
   max_memory = 128.GB
   max_cpus = 16
   max_time = 240.h
+  igenomes_base = 's3://ngi-igenomes/igenomes/'
```

Comment:
Does that not have an impact on people without s3 access?

Reply:
No - it's public so as long as you have an internet connection, nextflow should be able to download the refs I think.

Reply:
This is cool then - thanks for clarifying!